packages/worker/src/workflow/bundler.ts

```diff
@@ -11,6 +11,12 @@ import { DefaultLogger, Logger } from '../logger';
 
 export const allowedBuiltinModules = ['assert'];
 export const disallowedBuiltinModules = builtinModules.filter((module) => !allowedBuiltinModules.includes(module));
+export const disallowedModules = [
+  ...disallowedBuiltinModules,
+  '@temporalio/activity',
+  '@temporalio/client',
+  '@temporalio/worker',
```

Comment:
Maybe add non-workflow-common, the opentelemetry client and worker libs and most of the testing stuff?

Reply:
👍 I like when I remember to use `jq`

```
$ cat packages/*/package.json | jq .name
"@temporalio/activity"
"@temporalio/client"
"@temporalio/common"
"@temporalio/core-bridge"
"@temporalio/create"
"@temporalio/docs"
"@temporalio/interceptors-opentelemetry"
"@temporalio/internal-non-workflow-common"
"@temporalio/internal-workflow-common"
"temporalio"
"@temporalio/proto"
"@temporalio/test"
"@temporalio/testing"
"@temporalio/worker"
"@temporalio/workflow"
```